presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PredicatePushDown.java

```diff
@@ -81,6 +81,7 @@
 import static com.facebook.presto.sql.planner.DeterminismEvaluator.isDeterministic;
```

Comment:
can you please copy PR description into commit message. Current commit message header is too long, please see that it even wraps in github.